apps/server/src/modules/learnroom/learnroom.module.ts

```diff
@@ -29,11 +30,10 @@ import {
 	RoomsService,
 } from './service';
 import { CommonCartridgeFileValidatorPipe } from './utils';
-import { BoardNodeRepo } from '../board/repo';
 
 @Module({
 	imports: [
-		BoardModule,
+		forwardRef(() => BoardModule),
```

Comment:
You add it to avoid dependency cycles? If yes, then please note that you remove it in the account module refactoring.

Reply:
Are you sure if this will be the case? We can try, but i think those cycles are seated deeper in general